tests/test_connection.py

```diff
@@ -213,6 +213,31 @@ def test_invalid_method_name(self):
         with self.assertRaises(SyncanoValueError):
             self.connection.make_request('INVALID', 'test')
 
+    @staticmethod
+    def _create_test_file():
```

Comment:
Maybe we should use here tempfile?

https://docs.python.org/2/library/tempfile.html